parsers/test/test_MD.py

```diff
@@ -121,13 +103,12 @@ def test_fetch_exchange_forecast_live(snapshot, fixture_session_mock, neighbour)
     )
 
     assert snapshot == fetch_exchange_forecast(
-        ZoneKey("MD"), ZoneKey(neighbour), session=session
+        ZoneKey("MD"), ZoneKey(neighbor), session=session
     )
 
 
 @pytest.mark.parametrize("neighbour", ["RO", "UA"])
```

Comment:
Neighbour(UK) remains used here but changed elsewhere to neighbor(USA).

Reply:
Ah, I'll align them in the morning then merge this.